packages/ui-components/src/Icon/Icon.tsx

```diff
@@ -15,45 +15,51 @@ export type IconSize =
   | "x-large"
   | "xx-large";
 
-export type IconTint =
-  | "blue"
-  | "green"
-  | "red"
-  | "white"
-  | "neutral"
-  | "inherit"
-  | "orange";
+export type IconStyle =
```

Comment:
For consistency we should call this IconIntent
```suggestion
export type IconIntent =
```